storage/client/client.go

```diff
@@ -1504,15 +1504,24 @@ func (c *StorageClient) RuntimeAccount(ctx context.Context, address staking.Addr
 		&totalSent,
 		&totalReceived,
 		&a.Stats.NumTxns,
-	); err != nil {
-		return nil, wrapError(err)
-	}
-	a.Stats.TotalSent, err = common.NumericToBigInt(totalSent)
-	if err != nil {
-		return nil, wrapError(err)
-	}
-	a.Stats.TotalReceived, err = common.NumericToBigInt(totalReceived)
-	if err != nil {
+	)
+
+	switch err {
+	case nil:
+		a.Stats.TotalSent, err = common.NumericToBigInt(totalSent)
+		if err != nil {
+			return nil, wrapError(err)
+		}
```

Comment:
Is this needed? The idea is that we should be able to read directly into `a.Stats.TotalSent`, and the conversion will happen automatically because we implement the `NumericDecoder` pgx interface: https://github.com/oasisprotocol/nexus/blob/f6af89d1101fd6c983ade51755a44dee0b6d0597/common/types.go#L112-L121

Reply:
Ah you are right, it's unnecessary. (The comment on line 679 points out that we only need to do this conversion when reading into a pointer). Updated!

Reply:
Thanks. FWIW, that comment is probably outdated. It refers to `big.Int`, whereas we now use `BigInt`, and one of the reasons for introducing a custom type was that we can support pgx serde.